functional-samples/cookbook.wasm-helloworld-print-nomodule/wasm/src/lib.rs

```diff
@@ -0,0 +1,25 @@
+extern crate wasm_bindgen;
```

Comment:
I think we can remove this line: https://doc.rust-lang.org/edition-guide/rust-2018/path-changes.html#no-more-extern-crate